app/styles/newcomponents/chart-bar.scss

```diff
@@ -1,4 +1,4 @@
-.chart-container {
+.ilios-chart {
```

Comment:
Since this is in the chart-bar file I think it should probably by `.chart-bar {`